djangoBackend/settings.py

```diff
@@ -21,7 +21,7 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get('SECRET_KEY')
+SECRET_KEY = 'SECRET_KEY'
```

Comment:
Rever this back :) It's easier to set a random secret key in your environment variables so that you don't have to do this every time.

Reply:
I reverd it.